scipy/special/orthogonal.py

```diff
@@ -1386,6 +1388,26 @@ def gegenbauer(n, alpha, monic=False):
     :math:`[-1,1]` with weight function :math:`(1 - x^2)^{(\alpha -
     1/2)}`.
 
+    Examples
+    --------
+    >>> from scipy import special
+    >>> import matplotlib.pyplot as plt
+    >>> import numpy as np
+
+    >>> p = special.gegenbauer(3, 0.5, monic=False)
+    >>> p
+    poly1d([ 2.5,  0. , -1.5,  0. ])
+    >>> p(1)
+    1.0
+    >>> x = np.linspace(-3, 3, 400)
+    >>> y = p(x)
+
+    >>> plt.plot(x, y)
+    >>> plt.title("Gegenbauer (ultraspherical) polynomial of degree 3")
+    >>> plt.xlabel("x")
+    >>> plt.ylabel("G_3(x)")
+    >>> plt.show()
+
```

Comment:
The whitespace breaks the code blocks:
![image](https://user-images.githubusercontent.com/6570539/118593250-39d3fb00-b75c-11eb-871f-0a292384dd6a.png)

Is this intentional? One use for breaks would be to add commentary of what's going on and/or to break up the code as you might break paragraphs in a document.

Reply:
I felt the line breaks helped the code be more easily read (give it some "breathing room"). 
Also, just noticed based on the doc https://www.sciencedirect.com/topics/mathematics/gegenbauer-polynomial it might make more sense to change `x = np.linspace(-1, 1, 400)`

Reply:
I second Matt here. It would be informative to give a bit of context. Just a sentence even.

Reply:
Ok, can add a bit more context.

Reply:
Added context in [this commit](https://github.com/scipy/scipy/pull/14067/commits/0e14f738809ea817b86a1e9392394361e1772822)